src/adaptors/star_citizen.py

```diff
@@ -121,6 +123,51 @@ def add_device(self, option):
 
     def process_name(self, name):
         helper.log("Bind Name: {}".format(name), "debug")
+        # Force some Labels, this ideally need to be declared elsewhere or from an external file
+        customLabels = {
```

Comment:
Can we define these in outer scope at class level please? Overall format is all cool.

Reply:
I'll take alook for this, it's being redefined on every loop and it's not a good practice.

Reply:
Yeah they should be consts but then all good.

Reply:
moved outside of the function defenition at the class level.
Will add more label shortly